verilog/analysis/checkers/BUILD

```diff
@@ -46,6 +46,7 @@ cc_library(
         ":module-begin-block-rule",
         ":module-filename-rule",
         ":module-instantiation-rules",
+        ":dff-name-style-rule",
```

Comment:
Can you put this in here sorted to make it easier to find ? Between `:create-object-name-match-rule` and `:disable-statement-rule`